apps/vor/frontend/src/api/guardians/useGetGuardian.ts

```diff
@@ -1,14 +1,14 @@
 import { QueryResult, useQuery } from "react-query"
 import { getApi } from "../fetchApi"
 
-interface Guardians {
+interface Guardian {
   id: string
   name: string
   email: string
   note: string
   phone: string
 }
-export const useGetGuardian = (guardianId: string): QueryResult<Guardians> => {
-  const fetchGuardian = getApi<Guardians>(`/guardians/${guardianId}`)
+export const useGetGuardian = (guardianId: string): QueryResult<Guardian> => {
+  const fetchGuardian = getApi<Guardian>(`/guardians/${guardianId}`)
   return useQuery(["guardian", { guardianId }], fetchGuardian)
```

Comment:
try replacing `{ guardianId }` with just `guardianId`, i think that is why the patch doesn't automatically updates current value, the id doesn't match.